widgetssdk/src/main/java/com/glia/widgets/view/head/ActivityWatcherForChatHead.kt

```diff
@@ -24,12 +34,16 @@ internal class ActivityWatcherForChatHead(
         controller.setWatcher(this)
     }
 
+    companion object {
```

Comment:
please move it to the end of the class 
https://kotlinlang.org/docs/coding-conventions.html#class-layout

Reply:
Yes, thank you. Moved.